src/it/java/teammates/it/storage/sqlapi/UsersDbIT.java

```diff
@@ -44,6 +47,12 @@ public void setUp() throws Exception {
         course = new Course("course-id", "course-name", Const.DEFAULT_TIME_ZONE, "institute");
         coursesDb.createCourse(course);
 
+        section = new Section(course, "test-section");
+        course.addSection(section);
+        team = new Team(section, "test-section");
```

Comment:
Let's make the team name `"test-team"` to avoid any potential future confusion

Reply:
changed!